kartothek/io/dask/common_cube.py

```diff
@@ -489,19 +516,23 @@ def _multiplex_store_dataset_from_partitions_flat(
     result = {}
     for k, v in dct.items():
         if update:
+            print(
```

Comment:
either remove the debug prints or replace them with logging

Reply:
Ups, this is a leftover from my local debugging sessions. Removed it via a new commit.